dc/council/code/titles/1/sections/1-1001.16.xml

```diff
@@ -235,29 +235,43 @@
     <para>
       <num>(1)</num>
       <text>Upon qualification of an initiative measure, the Board shall place on the ballot the serial number of the initiative and its short title and summary statement in substantially the following form:</text>
-      <para>
-        <heading>INITIATIVE MEASURE No. (SHORT TITLE)</heading>
-      </para>
+      <text>
+        <strong>INITIATIVE MEASURE No.</strong>
+        <br/>
+        <strong>(SHORT TITLE)</strong>
+        <br/>
+        <strong>(SUMMARY STATEMENT)</strong>
+        <br/>
+        <strong>FOR Initiative Measure No.</strong>
```

Comment:
Well, sort of ... see [Division I.docx](https://github.com/openlawlibrary/scratch/blob/master/code-2016-07/Division%20I.docx) for that section. The text appears as so:

>INITIATIVE MEASURE No. 
>(SHORT TITLE) 
>(SUMMARY STATEMENT) 
>FOR Initiative Measure No. 
>AGAINST Initiative Measure No.

on Page 1526, except that it is centered. I can push a fix to center it.